spark/v3.3/spark/src/jmh/java/org/apache/iceberg/spark/data/parquet/SparkParquetReadersFlatDataBenchmark.java

```diff
@@ -155,6 +155,7 @@ public void readUsingSparkReader(Blackhole blackhole) throws IOException {
             .set("spark.sql.parquet.binaryAsString", "false")
             .set("spark.sql.parquet.int96AsTimestamp", "false")
             .set("spark.sql.caseSensitive", "false")
+            .set("spark.sql.parquet.fieldId.write.enabled", "false")
```

Comment:
this flag was introduced with Spark 3.3.0 and `SparkToParquetSchemaConverter` requires this flag to be set